global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "8.0.110",
```

Comment:
We can bump to the same SDK as `main` branch -- provided it doesn't incur a bunch of warnings or errors (and you don't want to address them)